cms/djangoapps/contentstore/tests/test_import.py

```diff
@@ -283,3 +283,54 @@ def _verify_split_test_import(self, target_course_name, source_course_name, spli
         }
 
         self.assertEqual(remapped_verticals, split_test_module.group_id_to_child)
+
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_video_components_present_while_import(self, store):
```

Comment:
Nevermind.

Actually, in looking at the bug closer, it seems the issue is that the course does get imported, but the video module is not created.  Also, this is an issue only if the `target_id` of the course is different when reimporting.

Did you run the test without the fix and make sure it failed?